db/migrate/20181001191550_create_book_suggestions.rb

```diff
@@ -0,0 +1,16 @@
+class CreateBookSuggestions < ActiveRecord::Migration[5.2]
+  def change
+    create_table :book_suggestions do |t|
+      t.string :synopsis, default: ''
+      t.float :price, default: 0.0
+      t.string :author, null: false, default: ''
+      t.string :title, null: false, default: ''
+      t.string :link, null: false, default: ''
+      t.string :publisher, null: false, default: ''
+      t.string :year, null: false, default: ''
+      t.references :user, foreign_key: true
```

Comment:
We could add an idex for this reference

Reply:
@facucompe it is indexed 👍 

In Rails 5, the index option defaults to true for add_reference:

https://api.rubyonrails.org/v5.2/classes/ActiveRecord/ConnectionAdapters/SchemaStatements.html#method-i-add_reference